src/cli.ts

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+
+import createJob, { JobDescription } from './create-job';
+import which from 'which';
+import yargs from 'yargs/yargs';
+import { hideBin } from 'yargs/helpers';
+import logger from './logger';
+
+async function run() {
+
+    const argv = await yargs(hideBin(process.argv))
+        .scriptName('autovmaf')
+        .command(['* <source>'], 'run autovmaf for videofile source', (yargs) => {
+            return yargs
+                .positional('source', { type: 'string', describe: 'SOURCEFILE', demandOption: true })
+        })
+        .options({
+            resolutions: { type: 'string', description: 'List of resolutions, ie 1920x1080,1280x720...' },
+            bitrates: { type: 'string', description: 'List of bitrates, ie 800k,1000k,...' },
+            name: { type: 'string', description: 'Name for this autovmaf run', default: 'MyVMAFMeasurements'},
+            models: {type: 'string', description: 'List of VMAF Models to use', default: 'HD'}
+        })
+        .parse();
+
+    const reference: string = argv.source;
+    const {pythonPath, ffmpegPath, easyVmafPath} = await getExecutablePaths();
+    
+    const pipeline = {
+        local: {
+            ffmpegPath,
+            pythonPath,
+            easyVmafPath,
+            ffmpegEncoder: "libx264",
+            ffmpegOptions: {
+                preset: "veryfast"
```

Comment:
Is this something we need to set? `veryfast` just seems... ugly 😄

 Ideally one should be able to pass in custom ffmpeg parameters at some point. Not a requirement now, though.

Reply:
I agree the preset should not be set by default. I will add support for custom ffmpeg parameters, should be a quick fix I think.